locallib.php

```diff
@@ -300,7 +300,8 @@ function theme_boost_union_get_staticpage_link($page) {
  */
 function theme_boost_union_get_staticpage_pagetitle($page) {
     // Get the configured page title.
-    $pagetitleconfig = get_config('theme_boost_union', $page.'pagetitle');
+    $pagetitleconfig = format_string(get_config('theme_boost_union', $page.'pagetitle'), false,
```

Comment:
Why are you setting the second parameter (striplinks) from the default (true) to false? The pagetitle configuration does not contain any links at all, the link will be composed later in mustache from the pagetitle and the page's URL.
In my opinion, there is no need to let any links (which the admin might have set in the pagetitle configuration by mistake or maliciously) pass through.